### ISSUE TITLE
ERROR invalid character '"' after object key:value pair

### ISSUE THREAD

**Issue by Animal-UK** [qdm12/ddns-updater#228]
Hello,

After deploy with docker-compose I have this error:

```
Running version latest built on 2021-08-09T14:56:55Z (commit a4af81d)
 Need help? https://github.com/qdm12/ddns-updater/issues/new
Email? quentin.mcgaw@gmail.com
Slack? Join from the Slack button on Github
Help me? https://github.com/sponsors/qdm12
2021/08/10 13:07:05 ERROR invalid character '"' after object key:value pair
2021/08/10 13:07:05 INFO Shutdown successful
```

My docker compose:

```yml
    DDNS-Updater:
        container_name: ddns-updater
        environment:
            - 'TZ=Europe/London'
            - 'DATADIR=/updater/data'
            - 'PERIOD=5m'
            - 'HTTP_TIMEOUT=10s'
            - 'BACKUP_DIRECTORY=/updater/data/backups'
            - 'BACKUP_PERIOD=72h0m' # 0 to disable
        volumes:
            - '/share/Container/container-station-data/appdata/DDNS-Updater/data:/updater/data'
        network_mode: bridge
        ports:
            - '8001:8000'
        restart: unless-stopped
        image: qmcgaw/ddns-updater:latest
```

thanks in advance

**Comment by qdm12** [comment]
Hi there!

> invalid character '"' after object key:value pair

I'm not sure, but it's most likely the JSON file that is incorrect. Maybe you have an extra `"` after a key value pair?

**Comment by Animal-UK** [comment]
Yeah i thought so too and checked but this isnt any :(

**Comment by qdm12** [comment]
Fixed in 41e61ddaa65b8c7502a7504584967f3c60d6e367

However, the CI is broken, so it should build sometime once this is fixed:

> Incident with GitHub Actions. We hit an issue with the main request routing service which is causing errors with running jobs, and retrieving logs/artifacts. The team has identified the cause and is actively mitigating.
